js/ui/radio_group/radio_group.js

```diff
@@ -299,7 +303,9 @@ var RadioGroup = Editor.inherit({
         this.itemElements().each((function(_, item) {
             var $item = $(item);
             var itemValue = this._valueGetter($item.data(ITEM_DATA_KEY));
-            $item.toggleClass(RADIO_BUTTON_CHECKED_CLASS, this._isValueEquals(itemValue, selectedValue));
+            var isValueEquals = this._isValueEquals(itemValue, selectedValue);
+            $item.toggleClass(RADIO_BUTTON_CHECKED_CLASS, isValueEquals);
```

Comment:
You can use chaining for this calls. And start each new chained operation with a new line, please. It would more readable
```
$item
  .toggleClass
  .find
...